service.go

```diff
@@ -2,6 +2,19 @@ package libnetwork
 
 import "net"
 
+type serviceTarget struct {
+	name   string
```

Comment:
`name` here is the backing container (ie: task name) or the host name which will be different for each instance.